Bottom: af22931162739b6ef27bde9ea60cb16cfbb13900
Top:    3c3f0f8a447e86b0bbd6e61d3ab5e43a37ac7ce0
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-14 15:18:34 +0200

Refresh of add-a-calculator-value-object

---

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 51058b6..2f61c76 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,5 +1,4 @@
 add_subdirectory(modules)
-
 add_executable(rpn
 	rpn.cpp
 )
